src/main/scala/com/johnsnowlabs/nlp/annotators/sbd/pragmatic/PragmaticContentFormatter.scala

```diff
@@ -161,12 +161,12 @@ class PragmaticContentFormatter(text: String) {
     */
   def formatPunctuations: this.type = {
 
-    val factory = new RuleFactory(MATCH_ALL, PROTECT_FROM_BREAK)
+    val factory = new RuleFactory(MATCH_ALL, APPEND_WITH_SYMBOL)
     // http://rubular.com/r/mQ8Es9bxtk
       //continuous punctuations
-      .addRule(new RegexRule("(?<=\\S),(!|\\?){3,}(?=(\\s|\\z|\\$))", "formatPunctuations-continuous"))
+      .addRule(new RegexRule("(?<=\\S)[!\\?]+(?=\\s|\\z|$)", "formatPunctuations-continuous"))
```

Comment:
This new regex no longer checks for a strict sequence of punctuations but just for the presence of punctuations char. Seems that the regex description and comments are not related anymore.

Reply:
Thank you, you are correct, I'll take a look